examples/crowdfunding/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "crowdfunding-application",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "chai": "^4.2.0"
```

Comment:
how about adding `chai` to the root package.json?

Reply:
removed it, we are not using this in example